src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Color.values().length);
+        return Color.values()[index].toString();
```

Comment:
```suggestion
        return Color.values()[index].name();
```
If you want to get string value of enum you should use name() method
toString() can be overridden and would return the same value for all enum values

Reply:
Wow, thanks for such hint